interfaces/builtin/steam_support.go

```diff
@@ -186,6 +186,10 @@ mount options=(rw, rbind) /bindfile* -> /newroot/run/host/container-manager,
 # Allow mounting Nvidia drivers into the sandbox
 mount options=(rw, rbind) /oldroot/var/lib/snapd/hostfs/usr/lib/@{multiarch}/** -> /newroot/var/lib/snapd/hostfs/usr/lib/@{multiarch}/**,
 
+mount options=(rw, rbind) /oldroot/var/lib/snapd/hostfs/usr/share/** -> /newroot/**,
+mount options=(rw, rbind) /oldroot/var/lib/snapd/hostfs/ -> /newroot/var/lib/snapd/hostfs/,
```

Comment:
It would be worth leaving a comment about what information PV is attempting to get from these locations. Is this from following symlink trees in `/var/lib/snapd/lib/gl` to the Nvidia user space drivers?

Reply:
Yes, a comment would be useful here.